koku/api/report/test/test_queries.py

```diff
@@ -586,28 +590,84 @@ def test_apply_group_null_label_empty_string(self):
         out_data = handler._apply_group_null_label(data, groups)
         self.assertEqual(expected, out_data)
 
-    # FIXME: need test for _apply_group_by
```

Comment:
These are all already tested by our api tests

Reply:
Thanks for cleaning these up.